tests/specs/benchmarks/verification.k

```diff
@@ -211,6 +211,10 @@ module VERIFICATION
 
     rule #ceil32(X) => X requires X modInt 32 ==Int 0
 
+    rule 0 <=Int #ceil32(X) -Int X => true requires 0 <=Int X [simplification]
```

Comment:
equivalent to 2 lemmas below. Lemma below likely just requires [simplification]

Reply:
Which lemmas specifically? I don't see them.

Reply:
@denis-bogdanas I've done as you say here, and at least for the proofs here, they still pass with your suggested change, so I've pushed it up.